src/css/components/_step-by-step.scss

```diff
@@ -0,0 +1,47 @@
+.step-by-step {
+  display: flex;
+  gap: var(--space--base);
+  position: relative;
+  text-align: center;
+
+  &::before {
+    background-color: var(--color--text);
+    content: "";
+    display: block;
+    height: var(--border-width--thick);
+    left: 0;
+    position: absolute;
+    top: 50%;
+    width: 100%;
+    z-index: 0;
+  }
+}
+
+.step-by-step__item {
+  align-items: center;
+  background-color: var(--color--background-primary);
+  border: var(--border--base);
+  border-radius: var(--border-radius--base);
+  display: flex;
+  flex-direction: column;
+  padding: var(--space--base);
+  z-index: 1;
+  //width: clamp(9rem, 14vw, 15rem);
+  width: 15rem;
+}
+
+.step-by-step__title {
+  font-weight: var(--font-weight--bold);
+}
+
+@media only screen and (max-device-width: 400px) { 
```

Comment:
In scss we can nest media queries in the selectors themselves! Let's throw this within the `step-by-step` selector. We should also use `em`s for media querie units (in lieu of pixels for proportional sizing). And we can use `min-width` and `max-width` in lieu of `max-device-width` and `min-device-width`.

One last thing to note is with media queries, it's best practice to work upwards from the smallest device. So we set rules for the smaller device as the default and then the larger breakpoint is within a breakpoint. The idea being that designing for the smallest device will work on anything and should be default, but designing for large doesn't work the same way. So it'd be something like:

```css
.step-by-step {
  // styles for smaller viewport

  @media only screen and (min-width: 20em) {
    // styles for larger viewport
  }
}

Reply:
I moved the media query and nested it properly (i hope!). I opted for the max-width approach for now, if that's okay, because of ease right now. Thanks!